mwdb/model/user.py

```diff
@@ -154,12 +154,15 @@ def create(
             db.session.commit()
         return user
 
-    def _generate_token(self, fields, scope, expiration):
+    def _generate_token(self, fields, scope, expiration, **kwargs):
```

Comment:
Uh, that dict concatenation is a bit ugly :sweat_smile:. I would rewrite it a bit:
```suggestion
    def _generate_token(self, user_fields, scope, expiration, **extra_fields):
        token_data = {"login": self.login, **extra_fields}
        for field in user_fields:
           token_data[field] = getattr(self, field)
```